parsons/civis/civisclient.py

```diff
@@ -105,7 +105,7 @@ def table_import(self, table_obj, table, max_errors=None,
                 the future object.
         `Returns`
             ``None`` or ``civis.CivisFuture``
-        """ # noqa: E501,E261
+        """  # noqa: E501,E261
```

Comment:
I'm confused about this one. Does current guidance call for a double space before a comment following a comment block? Should this be on a separate line? It seem oddly structured, but I'm guessing I just am not familiar with the governing style rules.

Reply:
My VS code flake8 auto-linter added the space